modules/ROOT/pages/_partials/metrics/7_monitoring-sync.adoc

```diff
@@ -1,18 +1,29 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
 
+[id='monitoring-with-sync-{context}']
+= Monitoring the {sync-service} Service
 
```

Comment:
@wei-lee will this still be valid with new data sync?

Reply:
yes, there will be a Grafana dashboard available for each sync app